src/epics/favorites.js

```diff
@@ -59,11 +59,7 @@ export const saveNewFavorite = (action$, store) =>
             name
                 ? [
                       setMapProps({ id, name, description }),
-                      setMessage(
-                          `${i18n.t('Favorite')} "${name}" ${i18n.t(
-                              'is saved'
-                          )}.`
-                      ),
+                      setMessage(i18n.t('Map "{{name}}" is saved.', { name })),
```

Comment:
This message is used twice, keep it as a `const` instead? Like `const getSavedMessage = name => i18n.t('Map "{{name}}" is saved.', { name })` and `setMessage(getSavedMessage(name))`. Will reward 100 DRY points 😃

Reply:
Fixed!